content/chainguard/administration/manage-chainctl-config.md

```diff
@@ -58,39 +58,55 @@ Global Flags:
 Use "chainctl config [command] --help" for more information about a command.
 ```
 
-The default `chainctl` config is similar to the following.
+To view your current `chainctl` config, run:
 
+```shell
+chainctl config view
 ```
-Base Config file: none
+
+You'll receive output similar to this:
+
+```shell
+# Base Config file: /home/erika/.config/chainctl/config.yaml
+auth:
+    mode: browser
+    device-flow: ""
 default:
     active-within: 24h0m0s
+    autoclose: true
+    autoclose-timeout: "10"
+    group: ""
+    identity-provider: ""
+    org-name: ""
+    skip-auto-login: false
+    skip-version-check: false
+    social-login: google-oauth2
+    use-refresh-token: true
 output:
     color:
-      fail: '#ff0000'
-      pass: '#00ff00'
-      warn: '#ffa500'
-    records:
-      notable: []
+        fail: '#ff0000'
+        pass: '#00ff00'
+        warn: '#ffa500'
+    silent: false
 platform:
     api: https://console-api.enforce.dev
     audience: https://console-api.enforce.dev
     console: https://console.enforce.dev
     issuer: https://issuer.enforce.dev
     registry: https://cgr.dev
-    timestamp-authority: https://tsa.enforce.dev
 ```
 
 The full documentation for the `chainctl config` command is available on the relevant [reference page](/chainguard/chainctl/chainctl-docs/chainctl_config/).
- 
+
 ## Edit chainctl config
 
-You can edit the `chainctl` config directly with an editor. The following command will open a text editor (nano, by default) where you can edit the local `chainctl` config. 
+You can edit the `chainctl` config directly with an editor. The following command will open a text editor (`nano`, by default) where you can edit the local `chainctl` config.
```

Comment:
is it that `nano` is the default used by chainctl or is it that it will open up whatever the default editor on your system is?

Reply:
I am not quite sure, but I believe it's the system's default editor.